components/toast/toast.scss

```diff
@@ -75,6 +75,62 @@
             @include transition-timing-function(ease-in-out);
         }
 }
+
+.toast__hint {
+    left: 50%;
+    z-index: 3;
+
+    width: auto;
+    height: auto;
+    margin-top: 16px;
+    margin-left: 0;
+    margin-right: 0;
+    padding-top: 8px;
+    padding-bottom: 8px;
+    border: 1px solid var(--center-channel-color) inset;
```

Comment:
Apologies, I missed this one. This border doesn't seem to be applying. 'inset' doesn't seem to work. You can just go with:

```
border: solid 1px var(--center-channel-color-16);
```

Reply:
Border appearance is fixed, thanks a lot.